tardis/model/base.py

```diff
@@ -72,7 +72,14 @@ def __init__(self, velocity, homologous_density, abundance, time_explosion,
         self.v_boundary_outer = v_boundary_outer
         self.homologous_density = homologous_density
         self._abundance = abundance
+        self.decayed = False
         self.time_explosion = time_explosion
+
+        self.raw_abundance = self._abundance
```

Comment:
@vg3095 I think the decay is extremely quick - maybe just a property would be fine. Actually can you profile how quick?